### ISSUE TITLE
"Cannot enable security mitigations" after Windows Update

### ISSUE THREAD

**Issue by RigoLigoRLC** [M2Team/NanaZip#315]
After a recent Windows update (Current: Windows 11 Pro 22H2, 22621.1848) NanaZip (2.0.450.0 x64, from MS Store) would fail to enable security mitigations and pop a dialog box. Program would still launch normally.

![图片](https://github.com/M2Team/NanaZip/assets/49363666/41d0a253-06ec-4428-97c0-3bea349926ab)

Not sure what else I can further provide to assist debugging.

**Comment by RigoLigoRLC** [comment]
Note: launching NanaZip as administrator would not help so this might not be a privilege issue (or maybe I'm wrong). The Windows user is an admin user.

**Comment by MouriNaruto** [comment]
I cannot reproduce this issue on my Windows 11 Enterprise 22H2 Build 22621.1848 machine with NanaZip 2.0.450 x64 on Microsoft Store. So, I need more information about that.

Kenji Mouri

**Comment by Artoria2e5** [comment]
Happened to me on some boots, not on others. Hmm.

**Comment by AndromedaMelody** [comment]
It seems the bug of KB5028407?
https://support.microsoft.com/en-us/topic/kb5028407-how-to-manage-the-vulnerability-associated-with-cve-2023-32019-bd6ed35f-48b1-41f6-bd19-d2d97270f080

**Comment by RigoLigoRLC** [comment]
> Happened to me on some boots, not on others. Hmm.

Yeah I observed this too, after a reboot the issue is no longer reproducing. I'll come back with more info if I can get it to repro.

Also what I found before: launching from start menu / double clicking archives will bring out the error message, directly launching exe from Everything's search result won't.

**Comment by oxygen-dioxide** [comment]
This bug also happens to me.
I've set nanazip as the default program for opening zip archive. After downloading a zip file with firefox, I open the zip file from firefox and sometimes firefox freezes. (It doesn't show any error dialog, and the UI doesn't grey out by window, but it doesn't respond to mouse and keyboard.) After a long time, firefox returns alive and Nanazip shows the "Cannot enable security mitigations" dialog.

Windows 26016
Firefox 121.0
Nanazip 3.0.756.0

**Comment by oxygen-dioxide** [comment]
I found a way to stably reproduce this error dialog:
- Set Nanazip as the default program for opening zip file
- Launch Firefox, download a zip file
- Run `taskkill /im explorer.exe /F` in cmd
- Open the zip file you downloaded from firefox

**Comment by dinhngtu** [comment]
I believe this error has been fixed in #869? Feel free to reopen if problem persists.
